go.mod

```diff
@@ -6,13 +6,10 @@ require (
 	github.com/btcsuite/btcutil v0.0.0-20190425235716-9e5f4b9a998d
 	github.com/go-pg/migrations v6.7.3+incompatible
 	github.com/go-pg/pg v0.0.0-20190627115636-374b7dab11ff
-	github.com/jinzhu/inflection v1.0.0 // indirect
+	github.com/grpc-ecosystem/go-grpc-middleware v1.1.0
```

Comment:
plz make sure rebase master and tidy before merge

Reply:
ok